extensions/identity-hub/src/test/java/org/eclipse/dataspaceconnector/identityhub/store/IdentityHubInMemoryStoreTest.java

```diff
@@ -32,7 +32,7 @@ void addAndReadVerifiableCredential() {
         // Arrange
         var store = new IdentityHubInMemoryStore();
         var credentialsCount = FAKER.number().numberBetween(1, 10);
-        List<VerifiableCredential> credentials = Stream.generate(() -> VerifiableCredential.Builder.newInstance().id(FAKER.internet().uuid()).build())
+        var credentials = Stream.generate(() -> VerifiableCredential.Builder.newInstance().id(FAKER.internet().uuid()).build())
```

Comment:
nit: EDC tests use this sort of form instead. Though your code is equally valid and probably more readable, you might consider this for uniformity
```
IntStream.range(0, credentialsCount)
                .mapToObj(i -> VerifiableCredential.Builder.newInstance().id(FAKER.internet().uuid()).build())
```